dropwizard-servers/src/main/java/com/palantir/remoting1/servers/DropwizardTracingFilters.java

```diff
@@ -54,25 +54,26 @@ private DropwizardTracingFilters() {}
      * <p>
      * TODO(rfink) Is there a more stable way to retrieve IP/Port information?
      */
-    static void registerTracers(Environment environment, Configuration config, String tracerName) {
-        ServerTracer serverTracer = getServerTracer(extractIp(config), extractPort(config), tracerName);
+    static Brave registerTracers(Environment environment, Configuration config, String tracerName) {
```

Comment:
Eh, this kind of stuff makes me wonder if Brave is actually the right library for us.

I guess we can add our own Tracing object with static methods to add and complete traces that delegate to a Brave instance that DropwizardServers sets up. This gets us pretty close to just doing the impl ourselves, though.